extlib/reachability/Cargo.toml

```diff
@@ -0,0 +1,17 @@
+[package]
+name = "reachability"
+version = "0.0.1"
+edition = "2021"
+
+[features]
+default = []
+
+[dependencies]
+tikv-jemallocator = { version = "0.5.4", optional = true }
+clap = { version = "4.0.0", features = ["derive"] }
+simple_logger = { version = "4.3.3", features = ["stderr", "colors"], default-features = false }
+log = "0.4.17"
+snippets = { git = "https://github.com/fossas/lib-snippets", version = "0.2.0", features = ["lang-java-11"] }
+stable-eyre = "0.2.2"
```

Comment:
I'm worried that we are using too many error handling libraries.

Broker uses `error-stack`. Lernie and Sherlock use `this-error`. Reachability-toolkit uses `anyhow`.

Now we're adding `eyre` into the mix.

If there's a good reason to use `stable-eyre` instead of one of the other three, then that's fine. But I wanted to double-check

Reply:
I copied the dependencies and code structure from `berkelydb`, but I can modify it to recent rust code one's.

Reply:
Hmm. Maybe we should merge this as-is, but have a quick conversation about whether we want to standardize this a bit in standup tomorrow.

I don't think it blocks this PR; we can just make the change later if we decide to